scss/components/atoms/index.scss

```diff
@@ -0,0 +1,6 @@
+@charset 'UTF-8';
```

Comment:
After reading several documents about partial dependency management it turns out the the industry convention for roll up files is to use an index.scss file. Future versions of sass will automatically include these if not defined in the `@import` statements